### PR TITLE
Add OpenTF alpha test key

### PR DESCRIPTION
**Pull request by mbialon**

### CODE REVIEW

File: lambda/internal/providers/keys/opentf/20230915.asc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
```
Comment (cube2222) [inline lambda/internal/providers/keys/opentf/20230915.asc:2]: Is this empty line intentional?

Comment (mbialon) [reply lambda/internal/providers/keys/opentf/20230915.asc:2]: @cube2222 It's in the [spec](https://www.rfc-editor.org/rfc/rfc4880.html#section-6.2).

```
- Armor Headers

- A blank (zero-length, or containing only whitespace) line

- The ASCII-Armored data
```

Comment (cube2222) [reply lambda/internal/providers/keys/opentf/20230915.asc:2]: Ah, thanks for the explanation @mbialon!

**Review by cube2222 — COMMENTED** [review]
LGTM, but I'd like someone who's been working on the registry to approve.

**Review by Yantrio — APPROVED** [review]

**Pull request merged**